manifests/example-template-windows.yml

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-vm-template
+  annotations:
+    description: "OCP kubevirt windows VM template"
+    tags: "kubevirt,ocp,template,windows"
+  labels:
+    miq.github.io/kubvirt/os: windows-2016
+    miq.github.io/kubevirt-is-vm-template: true
```

Comment:
Onle label uses `kubevirt/os` the other `kubevirt-is-vm-template` - For the sake of consistency I'd switch to one way (either kubevirt- or kubevirt/)

I'd take the / separation, my 2ct.

Reply:
Problem arises from here: